Lambdas/GHHooks/Constants.swift

```diff
@@ -4,6 +4,7 @@ enum Constants {
     enum Channels: ChannelSnowflake {
         case logs = "1067060193982156880"
         case issueAndPRs = "1123702585006768228"
+        case release = "431926479752921098"
```

Comment:
Not a thing for this PR, of course, but as a matter of a _possible_ future TODO, does Discord's API allow for looking up these IDs versus just hardcoding them?

Reply:
Yeah but then we'll be hadcoding channel names instead of ids.

Reply:
Good point... I think I was conceptually reaching for "can this be configurable instead of forcing a rebuild and deploy to change them" ('cause we do that sooooooooooooooooooooo often and all 🤣)

Reply:
yeah it's suboptimal but not much we can do 😅
maybe moving all these to configuration files and making Penny read from those and ... but far from worth it.

Reply:
Yeah, agreed.